Tools/debug/CollectSystemInfo.ps1

```diff
@@ -88,6 +88,75 @@ function Export-EventLogs {
     }
 }
 
+function Ensure-DiskSpd {
+    $scriptDir = $null
+    try {
+        $scriptDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+    } catch {}
+    if (-not $scriptDir -or $scriptDir -eq '' -or -not (Test-Path $scriptDir)) {
+        $scriptDir = (Get-Location).Path
+    }
+
+    $diskspdExe = Join-Path $scriptDir 'diskspd.exe'
+    $zipPath = Join-Path $scriptDir 'DiskSpd.ZIP'
```

Comment:
@YanVugenfirer Do we need *pdb in the Zip archive, maybe remove it?

Reply:
Are those diskspd pdbs? Why do we need them? Are they needed for tracing?

Reply:
>Are those diskspd pdbs? 

Yes

>Why do we need them?

It was in release artifact the same as we do for drivers